hasura.planx.uk/metadata/tables.yaml

```diff
@@ -1612,6 +1612,26 @@
     - name: team
       using:
         foreign_key_constraint_on: team_id
+  insert_permissions:
```

Comment:
All fields have been altered to accept all, but I do feel an urge that it could only need to be ``id`` and ``team_id``. Let me know if the door is too wide here...

Reply:
I think for now it's fine to keep this as `INSERT` only for `platformAdmins` - it's a pretty limited user group and we'll always be able to adjust.